x-pack/plugins/lens/public/state_management/lens_slice.ts

```diff
@@ -709,6 +809,24 @@ export const makeLensReducer = (storeDeps: LensStoreDeps) => {
   });
 };
 
+function createFrameAPI(
```

Comment:
I know this PR is still in draft but couldn't we use the selector instead of this new function?

I already created a PR for this (https://github.com/elastic/kibana/pull/138430) suggestion.